README.md

```diff
@@ -45,7 +45,7 @@ Subsequent calls to CMake, e.g., when adding files to the build will not be nece
 
 ### Compiler choice
 CMake will default to your system's default compiler.
-To use a different one, call like `cmake -DCMAKE_C_COMPILER=clang -DCMAKE_CXX_COMPILER=clang++ ..` in a clean build directory.
+To use a different one, call like `cmake -DCMAKE_C_COMPILER=clang -DCMAKE_CXX_COMPILER=clang++ ..` in a clean build directory. See [dependencies](DEPENDENCIES.md) for supported compiler versions.
```

Comment:
Not your change, but where does the "like" come from?

Reply:
True, makes no sense.